common/navbar.js

```diff
@@ -59,7 +59,24 @@
             var parentNewTab = obj.newTab;
             // template the url
             // TODO: This is done here to prevent writing a recursive function (again) in `setConfigJSON()`
-            if (obj.url && isCatalogDefined(catalogId)) obj.url = ERMrest.renderHandlebarsTemplate(obj.url, null, {id: catalogId});
+            if (obj.url && isCatalogDefined(catalogId)) {
+                obj.url = ERMrest.renderHandlebarsTemplate(obj.url, null, {id: catalogId});
+                // only append pcid/ppid if link is to the same origin
+                // same origin urls will be relative, check for http instead(?)
+                // if http, make sure not same origin
+                if (obj.url.indexOf("http") == -1 || obj.url.indexOf($window.location.origin) != -1) {
```

Comment:
We should check for "chaise" urls. We don't want to add these query parameters to static urls and just the origin check is not enough.